doc/userguide/rules/xbits.rst

```diff
@@ -21,9 +19,13 @@ Notes
    ``track ip_dst``, if you want to match on the server response,
    you check it (``isset``) with ``track ip_src``.
 
--  To not alert, use ``noalert;``
+-  To not alert, use ``xbits:noalert;``, it can only be used as a standalone
+   option like ``flowbits``.
```

Comment:
I wonder if we should allow this.  You can already simply specify `noalert;` as an option.  So I think the following 2 rules are equivalent:
```
alert tcp any any -> any any (xbits:noalert; sid:1; rev:1;)
alert tcp any any -> any any (noalert; sid:1; rev:1;)
```

Reply:
Yeah. This was more to keep consistency with `flowbits` since we have `flowbits:noalert;`..